scripts/lint.sh

```diff
@@ -4,4 +4,4 @@ set -e
 set -x
 
 ruff pymysqlreplication
-black pymysqlreplication --check
+black pymysqlreplication --exclude pymysqlreplication/bitmap.py --check
```

Comment:
not exclude all bitmap.py

 apply # fmt: on is more better

Reply:
I think it would be helpful if you refer to [this document](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html)

_`It doesn’t reformat lines that end with # fmt: skip or blocks that start with # fmt: off and end with # fmt: on. # fmt: on/off must be on the same level of indentation and in the same block, meaning no unindents beyond the initial indentation level between them.`_